tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerTokenSerializer.cs

```diff
@@ -10,7 +10,7 @@
 using APIView;
 using APIViewWeb;
```

Comment:
Why are we linking to APIView?

Reply:
And APIViewWeb?

Reply:
Using codefilekind enum from apiVIew. And will using local enum and remove APIView dll in a seprate PR.